VideoAPI/VideoApi.DAL/Queries/GetAvailableRoomByRoomTypeQuery.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain;
+using VideoApi.Domain.Enums;
+
+namespace VideoApi.DAL.Queries
+{
+    public class GetAvailableRoomByRoomTypeQuery : IQuery
+    {
+        public VirtualCourtRoomType CourtRoomType { get; set; }
```

Comment:
same as ICommand comment - dont think we need setter

Reply:
done